persistence/src/main/scala/org/apache/pekko/persistence/snapshot/japi/SnapshotStore.scala

```diff
@@ -29,15 +30,15 @@ abstract class SnapshotStore extends SSnapshotStore with SnapshotStorePlugin {
   override final def loadAsync(
       persistenceId: String,
       criteria: SnapshotSelectionCriteria): Future[Option[SelectedSnapshot]] =
-    doLoadAsync(persistenceId, criteria).map(option)
+    doLoadAsync(persistenceId, criteria).asScala.map(option)
 
   override final def saveAsync(metadata: SnapshotMetadata, snapshot: Any): Future[Unit] =
-    doSaveAsync(metadata, snapshot).map(_ => ())
+    doSaveAsync(metadata, snapshot).asScala.map(_ => ())
```

Comment:
parasitic

Reply:
`ConstantFun.anyToUnit`

Reply:
there is no ConstantFun.anyToUnit

Reply:
I mean `  def scalaAnyToUnit[A]: A => Unit = unit`,
Sorry not check the code.

Reply:
I found `ConstantFun.unit` - I thought this was closer

Reply:
I see that is private